account_payment_promissory_note/readme/CONTRIBUTORS.rst

```diff
@@ -4,3 +4,9 @@
   * Alexandre Díaz
   * Carlos Roca
   * César A. Sánchez
+
+* `Avanzosc <https://www.avanzosc.es>`_:
+
+  * Berezi Amubieta
```

Comment:
You can't add 3 names for such simple migration.

Reply:
@pedrobaeza 
Is it correct now?
We need this merged asap
Thank you